searchbar/src/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import './styles.css';
 
 ReactDOM.render(
```

Comment:
when we combine these we'll have to name the dom element something unique so the reactdom.render knows where to put it in the proxy server.  I named both of mine Champagne in the dom and in the index.js file